### DISCUSSION TITLE
Can only take 1 picture to apply for GitHub Education

### DISCUSSION THREAD

**Discussion by superducky43** [community/community#144068] [GitHub Education]
### Hi everyone,

I have the problem that I can only submit one picture for the application. This is problematic because my student ID has two sides to it and my full name is on one side and the date of expiry is on the other.

**Comment by lslqtz** [comment]
Maybe, try combining into one image and uploading that?

**Comment by the-bipu** [comment]
you can submit single side, also if you want, then merge it in one frame and them upload it.
